src/resources/postcss/onboarding.pcss

```diff
@@ -0,0 +1,157 @@
+/*
+ * Onboarding CSS (Tours & Hints)
+ * -------------------------------------------------------------
+ * This file includes styles for the Onboarding Tours and Hints.
+ * These styles go on top of the ones from Intro.js
+ */
+
+/* Utilities */
+@import "utilities/_all.pcss";
+
+.introjs-tooltipbuttons {
+	border-top: none;
+}
+
+.introjs-bullets {
+	ul {
+		li a.active {
+			background-color: var(--color-accent-primary);
+		}
+
+		li:only-child {
+			display: none;
+		}
+	}
+}
+
+.introjs-hint-pulse {
+	background-color: transparent;
+}
+
+.introjs-hint-pulse,
+.introjs-hint:hover > .introjs-hint-pulse,
+.introjs-hint-dot {
+	border-color: var(--color-accent-primary);
+}
+
+.introjs-hintReference .tribe-onboarding__tooltip .introjs-tooltiptext p {
+	font-size: 14px;
+	margin-top: 0;
+}
+
+.introjs-hint:focus {
+	box-shadow: none;
+}
+
+.tribe-onboarding__tooltip {
+
+	.introjs-tooltipReferenceLayer:not(.introjs-hintReference) & {
+		box-sizing: border-box;
+		max-width: 450px;
+		min-width: 430px;
+	}
+
+	.introjs-button {
+		border: 0;
+		text-shadow: none;
+
+		&:focus {
+			box-shadow: none;
+		}
+	}
+
+	.introjs-prevbutton {
+		color: #c9c9c9;
```

Comment:
do we have a similar color variable we could probably use here?

Reply:
Will check that 👍